[ANDROID] App crashes on startup in dev env

The application crashes on startup in dev env with the next output ([commit which introduced the issue](https://github.com/status-im/status-mobile/commit/3bd4038089ebc4094728fd0fef7a5d5830653553)):

```
java.lang.RuntimeException: Unable to create application im.status.ethereum.MainApplication: java.lang.RuntimeException: java.io.IOException: could not delete: /data/user/0/im.status.ethereum.debug/lib-main/libbetter.so
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6596)
	at android.app.ActivityThread.access$1500(ActivityThread.java:230)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1911)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:224)
	at android.app.ActivityThread.main(ActivityThread.java:7551)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:995)
Caused by: java.lang.RuntimeException: java.io.IOException: could not delete: /data/user/0/im.status.ethereum.debug/lib-main/libbetter.so
	at com.facebook.soloader.SoLoader.init(SoLoader.java:214)
	at com.reactnativenavigation.NavigationApplication.onCreate(NavigationApplication.java:26)
	at im.status.ethereum.MainApplication.onCreate(MainApplication.java:70)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1189)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6591)
	... 8 more
Caused by: java.io.IOException: could not delete: /data/user/0/im.status.ethereum.debug/lib-main/libbetter.so
	at com.facebook.soloader.SysUtil.dumbDeleteRecursive(SysUtil.java:119)
	at com.facebook.soloader.UnpackingSoSource.deleteUnmentionedFiles(UnpackingSoSource.java:207)
	at com.facebook.soloader.UnpackingSoSource.regenerate(UnpackingSoSource.java:264)
	at com.facebook.soloader.UnpackingSoSource.refreshLocked(UnpackingSoSource.java:321)
	at com.facebook.soloader.UnpackingSoSource.prepare(UnpackingSoSource.java:414)
	at com.facebook.soloader.SoLoader.initSoSources(SoLoader.java:322)
	at com.facebook.soloader.SoLoader.init(SoLoader.java:200)
	at com.facebook.soloader.SoLoader.init(SoLoader.java:182)
	at com.facebook.soloader.SoLoader.init(SoLoader.java:212)
	... 12 more
```

OS: Android 10
Device: Redmi Note 8 Pro

```
05-04 10:43:04.396 16424 16509 D SoLoader: About to load: libflipper.so
05-04 10:43:04.397 16424 16509 D SoLoader: libflipper.so not found on /data/data/im.status.ethereum_foo.debug/lib-main
05-04 10:43:04.397 16424 16509 D SoLoader: libflipper.so found on /data/app/im.status.ethereum_foo.debug-y2fHDAGESke4ua-Zp7HqIw==/lib/arm64
05-04 10:43:04.397 16424 16509 D SoLoader: Not resolving dependencies for libflipper.so
05-04 10:43:04.399  7480  7537 W DimLayer: removeDimLayer
05-04 10:43:04.401  1534  1534 D EventBus: [1534, u0] send(AppTransitionFinishedEvent)
05-04 10:43:04.401  1534  1534 D EventBus: [1534, u0]  -> ForcedResizableInfoActivityController [0x2debc9b, P1] onBusEvent(AppTransitionFinishedEvent)
05-04 10:43:04.401  1534  1534 D EventBus: [1534, u0] onBusEvent(AppTransitionFinishedEvent) duration: 22 microseconds, avg: 25
05-04 10:43:04.401  1534  1534 D EventBus: [1534, u0] send(AppTransitionFinishedEvent)
05-04 10:43:04.402  7480  7537 E LauncherAnimationRunner: finish: mFinishRunnable.run
05-04 10:43:04.402  1534  1534 D EventBus: [1534, u0]  -> ForcedResizableInfoActivityController [0x2debc9b, P1] onBusEvent(AppTransitionFinishedEvent)
05-04 10:43:04.402  1534  1534 D EventBus: [1534, u0] onBusEvent(AppTransitionFinishedEvent) duration: 15 microseconds, avg: 25
05-04 10:43:04.402 16424 16509 W System.err: java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "_Unwind_Backtrace" referenced by "/data/app/im.status.ethereum_foo.debug-y2fHDAGESke4ua-Zp7HqIw==/lib/arm64/libflipper.so"...
05-04 10:43:04.402 16424 16509 W System.err: 	at java.lang.Runtime.load0(Runtime.java:938)
05-04 10:43:04.402 16424 16509 W System.err: 	at java.lang.System.load(System.java:1631)
05-04 10:43:04.402 16424 16509 W System.err: 	at com.facebook.soloader.SoLoader$1.load(SoLoader.java:405)
05-04 10:43:04.402 16424 16509 W System.err: 	at com.facebook.soloader.DirectorySoSource.loadLibraryFrom(DirectorySoSource.java:77)
05-04 10:43:04.402 16424 16509 W System.err: 	at com.facebook.soloader.DirectorySoSource.loadLibrary(DirectorySoSource.java:50)
05-04 10:43:04.402 16424 16509 W System.err: 	at com.facebook.soloader.ApplicationSoSource.loadLibrary(ApplicationSoSource.java:89)
05-04 10:43:04.402 16424 16509 W System.err: 	at com.facebook.soloader.SoLoader.doLoadLibraryBySoName(SoLoader.java:860)
05-04 10:43:04.402 16424 16509 W System.err: 	at com.facebook.soloader.SoLoader.loadLibraryBySoNameImpl(SoLoader.java:740)
05-04 10:43:04.402 16424 16509 W System.err: 	at com.facebook.soloader.SoLoader.loadLibraryBySoName(SoLoader.java:654)
05-04 10:43:04.402 16424 16509 W System.err: 	at com.facebook.soloader.SoLoader.loadLibrary(SoLoader.java:634)
05-04 10:43:04.402 16424 16509 W System.err: 	at com.facebook.soloader.SoLoader.loadLibrary(SoLoader.java:582)
05-04 10:43:04.402 16424 16509 W System.err: 	at com.facebook.flipper.android.EventBase.<clinit>(EventBase.java:19)
05-04 10:43:04.402 16424 16509 W System.err: 	at com.facebook.flipper.android.FlipperThread.run(FlipperThread.java:25)
05-04 10:43:04.402 16424 16509 E SoLoader: couldn't find DSO to load: libflipper.so caused by: dlopen failed: cannot locate symbol "_Unwind_Backtrace" referenced by "/data/app/im.status.ethereum_foo.debug-y2fHDAGESke4ua-Zp7HqIw==/lib/arm64/libflipper.so"... result: 0
05-04 10:43:04.404  2440  2521 D BluetoothUtils: blockedByLocationOff callingUid=1002
05-04 10:43:04.404  2440  2521 I BluetoothUtils: packagename is android.uid.bluetooth:1002 ,and its permission is false
05-04 10:43:04.404  1298  1317 I Timeline: Timeline: App_transition_stopped time:133230493
05-04 10:43:04.405  2440  2521 D BluetoothUtils: blockedByLocationOff callingUid=1002
05-04 10:43:04.405  2440  2521 I BluetoothUtils: packagename is android.uid.bluetooth:1002 ,and its permission is false
05-04 10:43:04.405  2440  2521 D BluetoothUtils: blockedByLocationOff callingUid=1002
05-04 10:43:04.405 16424 16509 E AndroidRuntime: FATAL EXCEPTION: FlipperConnectionThread
05-04 10:43:04.405 16424 16509 E AndroidRuntime: Process: im.status.ethereum_foo.debug, PID: 16424
05-04 10:43:04.405 16424 16509 E AndroidRuntime: java.lang.UnsatisfiedLinkError: couldn't find DSO to load: libflipper.so caused by: dlopen failed: cannot locate symbol "_Unwind_Backtrace" referenced by "/data/app/im.status.ethereum_foo.debug-y2fHDAGESke4ua-Zp7HqIw==/lib/arm64/libflipper.so"... result: 0
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at com.facebook.soloader.SoLoader.doLoadLibraryBySoName(SoLoader.java:918)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at com.facebook.soloader.SoLoader.loadLibraryBySoNameImpl(SoLoader.java:740)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at com.facebook.soloader.SoLoader.loadLibraryBySoName(SoLoader.java:654)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at com.facebook.soloader.SoLoader.loadLibrary(SoLoader.java:634)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at com.facebook.soloader.SoLoader.loadLibrary(SoLoader.java:582)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at com.facebook.flipper.android.EventBase.<clinit>(EventBase.java:19)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at com.facebook.flipper.android.FlipperThread.run(FlipperThread.java:25)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: Caused by: java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "_Unwind_Backtrace" referenced by "/data/app/im.status.ethereum_foo.debug-y2fHDAGESke4ua-Zp7HqIw==/lib/arm64/libflipper.so"...
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at java.lang.Runtime.load0(Runtime.java:938)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at java.lang.System.load(System.java:1631)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at com.facebook.soloader.SoLoader$1.load(SoLoader.java:405)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at com.facebook.soloader.DirectorySoSource.loadLibraryFrom(DirectorySoSource.java:77)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at com.facebook.soloader.DirectorySoSource.loadLibrary(DirectorySoSource.java:50)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at com.facebook.soloader.ApplicationSoSource.loadLibrary(ApplicationSoSource.java:89)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	at com.facebook.soloader.SoLoader.doLoadLibraryBySoName(SoLoader.java:860)
05-04 10:43:04.405 16424 16509 E AndroidRuntime: 	... 6 more
05-04 10:43:04.405 16424 16508 E AndroidRuntime: FATAL EXCEPTION: FlipperEventBaseThread
05-04 10:43:04.405 16424 16508 E AndroidRuntime: Process: im.status.ethereum_foo.debug, PID: 16424
05-04 10:43:04.405 16424 16508 E AndroidRuntime: java.lang.NoClassDefFoundError: <clinit> failed for class com.facebook.flipper.android.EventBase; see exception in other thread
05-04 10:43:04.405 16424 16508 E AndroidRuntime: 	at com.facebook.flipper.android.FlipperThread.run(FlipperThread.java:25)
05-04 10:43:04.406  2440  2521 I BluetoothUtils: packagename is android.uid.bluetooth:1002 ,and its permission is false
05-04 10:43:04.406  2440  2521 D BluetoothUtils: blockedByLocationOff callingUid=1002
05-04 10:43:04.406  2440  2521 I BluetoothUtils: packagename is android.uid.bluetooth:1002 ,and its permission is false
05-04 10:43:04.408 30791 30839 E MiuiMiHomeConnectController: error
```

--- a/android/app/src/main/java/im/status/ethereum/MainApplication.java
+++ b/android/app/src/main/java/im/status/ethereum/MainApplication.java
@@ -72,7 +72,7 @@ public class MainApplication extends NavigationApplication {
         OkHttpClientProvider.setOkHttpClientFactory(new StatusOkHttpClientFactory());
 
         WebView.setWebContentsDebuggingEnabled(BuildConfig.DEBUG_WEBVIEW == "1");
-        initializeFlipper(this, getReactNativeHost().getReactInstanceManager());
+        //initializeFlipper(this, getReactNativeHost().getReactInstanceManager());
     }
     /**
      * Loads Flipper in React Native templates. Call this in the onCreate method with something like

